bot.py

```diff
@@ -1,9 +1,19 @@
+"""
+This program is for building a twitter bot . Which can retweet and fav the tweets and suggest you something pretty awesome things to draw or paint. Which will inturn make you creative ;).
+HAPPY DRAWING 
+"""
+# Import the necessary modules... Don't forget to import our sweet tweepy ;)

```

Comment:
```suggestion
# Import the necessary modules... Don't forget to import our sweet tweepy ;)
```suggestion
# Import the necessary modules... Don't forget to import our sweet tweepy ;)

```
```